recipe/meta.yaml

```diff
@@ -11,9 +11,9 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: 1
+  number: 2
   script: "{{ PYTHON }} -m pip install . --no-deps -vv"
-  skip: true  # [win and py==27]
+  skip: true  # [win and py==27] or [ppc64le and py==27]
```

Comment:
This isn't really a valid selector ([spec](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html?highlight=selector#preprocessing-selectors)), and conda-build is actually making a ppc 2.7 build that then doesn't do anything? Pushing a fix.

Reply:
yeah - not really sure how to keep smithy from creating the job for 2.7..

Reply:
re-render fixed it..